README.md

```diff
@@ -101,12 +90,19 @@ This policy is to balance the desire for as much backwards compatibility as poss
 
 ## License
 
-[Apache 2.0](https://gitlab.com/Mordil/RediStack/blob/master/LICENSE.txt)
+[Apache 2.0][Apache License]
```

Comment:
Should we link to the local `LICENSE.md`?

Reply:
See at the bottom, where the links are actually located.